docs_user/modules/proc_pulling-configuration-from-a-tripleo-deployment.adoc

```diff
@@ -58,12 +65,109 @@ services:
 +
 Repeat this step for each {OpenStackShort} service that you want to disable or enable.
 
+Os-diff can pull configuration or command output from non-containerized servies such as the ovs-external-ids:
+
+----
+services:
+  ovs_external_ids:
+    hosts:
+      - standalone
+    service_command: "ovs-vsctl list Open_vSwitch . | grep external_ids | awk -F ': ' '{ print $2; }'"
+    cat_output: true
+    path:
+      - ovs_external_ids.json
+    config_mapping:
+      ovn-bridge-mappings: edpm_ovn_bridge_mappings
+      ovn-bridge: edpm_ovn_bridge
+      ovn-encap-type: edpm_ovn_encap_type
+      ovn-match-northd-version: ovn_match_northd_version
+      ovn-monitor-all: ovn_monitor_all
+      ovn-remote-probe-interval: edpm_ovn_remote_probe_interval
+      ovn-ofctrl-wait-before-clear: edpm_ovn_ofctrl_wait_before_clear
+----
+
+This service is not an Openstack service executed in a container, so the description and the behavior is different.
+It's important to correctly configure an SSH config file or equivalent for non standard services such as OVS.
+The ovs_external_ids is not a service which runs in a container and the ovs data is stored
+on each host of our cloud: controller_1/controller_2/...
+
+With the `hosts` key, os-diff will loop on each host provided and execute the command in `service_command` key:
+
+----
+    ovs_external_ids:
+        path:
+            - ovs_external_ids.json
+        hosts:
+            - standalone
+----
+
+The service_command is the command which provides the required information.
+It could be a simple cat from a config file. cat_output should be set to true
+if you want os-diff to get the output of the command and store the output in a file specified by the key path
+
+Then you can provide a mapping between in this case the EDPM CRD, and the ovs-vsctl output with config_mapping
+
+----
+        service_command: 'ovs-vsctl list Open_vSwitch . | grep external_ids | awk -F '': '' ''{ print $2; }'''
+        cat_output: true
+        config_mapping:
+            ovn-bridge: edpm_ovn_bridge
+            ovn-bridge-mappings: edpm_ovn_bridge_mappings
+            ovn-encap-type: edpm_ovn_encap_type
+            ovn-match-northd-version: ovn_match_northd_version
+            ovn-monitor-all: ovn_monitor_all
+            ovn-ofctrl-wait-before-clear: edpm_ovn_ofctrl_wait_before_clear
+            ovn-remote-probe-interval: edpm_ovn_remote_probe_interval
+----
+
+Then you can use this command to compare the values:
+
+----
+os-diff diff ovs_external_ids.json edpm.crd --crd --service ovs_external_ids
+----
+
+Lets take the example of checking the /etc/yum.conf on every hosts, you will have to put this statement in the config.yaml,
+lets call it yum_config:
+
+----
+services:
+  yum_config:
+    hosts:
+      - undercloud
+      - controller_1
+      - compute_1
+      - compute_2
+    service_command: "cat /etc/yum.conf"
+    cat_output: true
+    path:
+      - yum.conf
+----
+
```

Comment:
@matbu I started to format your new section as steps, but some of  this information does not appear to be steps that customers must do to complete the procedure. What information is necessary downstream?

CC: @jistr

Reply:
@klgill Yes it's not a necessary "step" but more some guide and help for the usage. I put that in place, but if it's sounds confusing we can drop it.